static/sass/_v1_pattern_phone_contact.scss

```diff
@@ -0,0 +1,22 @@
+// Resource box style patterns
+@mixin ubuntu-p-phone-contact {
+  @include p-phone-contact;
+}
+
+@mixin p-phone-contact {
+  .p-phone-contact {
+    padding: $sp-large;
+
+    &__wrapper {
+      background-image: url('#{$assets-path}00d6e773-contact_orange_phone.svg');
+      background-position: 10px;
+      background-repeat: no-repeat;
+      padding-left: 60px;
```

Comment:
I am sure we discussed a solution to image followed by text in a brochure theme workshop. It might have been a button along side some text. I can't remember what was the outcome. @barrymcgee @yaili do you remember?